srcpkgs/img/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'img'
+pkgname=img
+version=0.5.7
+revision=1
+build_style=go
+go_import_path=github.com/genuinetools/img
+go_build_tags="noembed seccomp"
+depends="libseccomp runc shadow"
+short_desc="Standalone, daemonless, unprivileged container image builder"
+maintainer="Noel Cower <ncower@gmail.com>"
+license="MIT"
+homepage="https://${go_import_path}"
```

Comment:
Just write out the homepage, otherwise e.g. void-updates will literally `https://${go_import_path}` as homepage.